im.nheko.Nheko.yaml

```diff
@@ -100,86 +100,90 @@ modules:
       - -Dvapi=false
       - -Dgtk_doc=false
       - -Dintrospection=false
-      # https://gitlab.gnome.org/GNOME/libsecret/-/issues/49
-      - -Dgcrypt=false
     sources:
       - commit: 3fe635e64efd4b8dbc9ec3548b0bc8034c7665c4
```

Comment:
You also need to update the commit, otherwise this will fail to download, see the CI failures.

Reply:
I think I forgot to update it before (anyway this should be fixed now)